vscode/src/test/suite/performance/open-jdk.test.ts

```diff
@@ -0,0 +1,135 @@
+import * as assert from 'assert';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as myExtension from '../../../extension';
+import { extensions, languages, Uri, window, workspace } from 'vscode';
+import { assertWorkspace, openFile, runShellCommand, waitCommandsReady } from '../../testutils';
+import { OPENJDK_CHECK_FILES_RESOLVES } from '../../constants';
+
+let lastFileSize = 0;
+let count = 0;
+const checkSymbolsResolved = (path: string): boolean => {
+    let d = languages.getDiagnostics(Uri.file(path));
+    const filterErrorsList = d.filter(el => el.severity == 0);
+    console.log("Filtered Errors List Length: " + filterErrorsList.length);
+    return filterErrorsList.length == 0;
+}
+
+const checkExtensionLoaded = (path: string): boolean => {
+    let d = languages.getDiagnostics(Uri.file(path));
+    return d.length != 0;
```

Comment:
Do add a comment on `constants.OPENJDK_CHECK_FILES_RESOLVES` that the files chosen must have some warnings or other diagnostic information/hints generated. Otherwise, this method would fail to detect that the extension loaded.